skbio/io/tests/data/qual_ws_line_between_records

```diff
@@ -0,0 +1,15 @@
+>seq1 desc1
```

Comment:
Can you add this file to the list of negatively-sniffed files in `SnifferTests`?